arm-asazure/2016-05-06-preview/swagger/asazure.json

```diff
@@ -0,0 +1,489 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-05-06-preview",
+    "title": "ASAzureWebAPI",
+    "description": "The Azure Analysis Services API provides a RESTful set of web services enables users to create, retrieve, update, and delete servers"
+  },
+  "host": "management.azure.com",
+  "basePath": "/webapi",
+  "schemes": [ "https" ],
+  "paths": {
+    "/resourceprovider/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers/{analysisServerName}": {
```

Comment:
This seems to be a new pattern to me
>"/resourceprovider/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers/{analysisServerName}

I have mostly seen paths starting with /subscriptions like this:
> "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers/{analysisServerName}

Can you please validate whether the current pattern is correct?